codecov.yaml

```diff
@@ -1,13 +1,15 @@
 coverage:
+  precision: 2
+  round: up
+  range: "45...70"
```

Comment:
This should be "45...100", right? :-)

Reply:
this is the color gradient applied to files (from red to green).  At our low code coverage this would make 70% or greater green.

Reply:
fair enough!